lib/galaxy/objectstore/__init__.py

```diff
@@ -289,6 +290,14 @@ def __init__(self, config, config_dict=None, **kwargs):
         extra_dirs.update({e["type"]: e["path"] for e in config_dict.get("extra_dirs", [])})
         self.extra_dirs = extra_dirs
 
+    def start(self):
+        """
+        Call all postfork function(s) here. These functions spawn a thread.
+        We register start(self) with app, so app starts the threads. Override
+        this function in subclasses, as needed.
+        """
+        return
```

Comment:
```suggestion
```
We don't need an explicit return here. Having the docstring is sufficient, I think.